forte/mrdsrg-spin-adapted/sadsrg.cc

```diff
@@ -82,8 +82,10 @@ SADSRG::~SADSRG() {
 void SADSRG::startup() {
     print_h2("Multireference Driven Similarity Renormalization Group");
 
-    // build Fock and cleanup JK in ForteIntegrals
-    build_fock_from_ints();
+    // build fock using ForteIntegrals and clean up JK
+    if (foptions_->get_bool("EMBEDDING_JKFOCK")) {
```

Comment:
To reproduce the old functionality, `embedding_jkfock` needs to be true in the case when embedding is off. If so, then this isn't an embedding keyword, but a DSRG one. Rename it. It looks like this one is 

Also, when would the user want the keyword on, and when does the user want it off? Should the user _ever_ set this manually?

Reply:
I think many of the options I added should not be used by user directly, instead, the aset2 driver should set it.
Maybe in document I should say those options are not meant to be touched.